ngraph/python/tests/test_onnx/model_zoo_preprocess.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 # provide ONNX Model Zoo commit hash ID to update:
-ONNX_SHA=7d9ae32726d872fe9143b7aede149508bdf0b30f
+ONNX_SHA=5f39d2534225a6f25012e578daf41ffd84bc00a1
```

Comment:
It will be updated again after https://github.com/onnx/models/pull/422 merge (missing mobilenet v2 test data)